package.json

```diff
@@ -38,6 +38,7 @@
     "redux-logger": "^3.0.6",
     "redux-persist": "^6.0.0",
     "redux-thunk": "^2.4.2",
+    "type-check": "^0.4.0",
```

Comment:
where do you use this library?

Reply:
Hmm... maybe I installed it for fixing 'CSS' property from StackOverflow. It didn't work.....:(